docs/core-concepts/index.md

```diff
@@ -0,0 +1,32 @@
+# Core Concepts
+
+These are some of the core concepts that you should be familiar with before going over the [Getting Started Guide](get-started/index.md).
+
+
+## GitOps
+GitOps is a way of managing your infrastructure and applications so that whole system is describe declaratively and version controlled(most likely in a git repository), and having an automated process that ensure that the deployed environment matches the state specified in a repository.
+
+For more information, take a look at [this documentation](https://gitops-community.github.io/kit/#what-is-gitops).
+
+## Git Source
+The git source represents a git repository in Flux.
+
+For more information, take a look at [this documentation](components/source/gitrepositories.md)
+
+## Kustomization
+The kustomization represents an application that Flux is supposed to reconcile in the cluster. It specifies a path in the git repository containing the `kustomization.yaml` for the application. If the repository contains only plain kubernetes manifests, a `kustomization.yaml` is generated for you. If there are non-kubernetes related files or manifest that you don't want to apply to the cluster, you can exclude it under the `spec.ignore` fields.
```

Comment:
```suggestion
The kustomization represents an local set of Kubernetes resources that Flux is supposed to reconcile in the cluster. [...]
```

I think the

> It specifies a path in the Git repository containing the `kustomization.yaml` for the application. If the repository contains only plain kubernetes manifests, a `kustomization.yaml` is generated for you. If there are non-kubernetes related files or manifest that you don't want to apply to the cluster, you can exclude it under the `spec.ignore` fields.

bit is technical information and the core concept. Instead, part of the information from "reconciliation" below can be moved here to describe the abstract.